src/gt4sd/domains/materials/scorer.py

```diff
@@ -327,8 +371,24 @@ def score(self, smiles: str) -> float:
         Returns:
             A score for the given SMILES
         """
+        print(Chem.QED.qed(Chem.MolFromSmiles(smiles)))
```

Comment:
remove print

Reply:
I have pushed a change for this, thanks :)